mep/people/forms.py

```diff
@@ -97,7 +97,9 @@ class MemberSearchForm(FacetForm):
         }),
         help_text='This filter will narrow results to show only members whose \
         library records are available.')
-    sex = FacetChoiceField(label='Sex')
+    sex = FacetChoiceField(label='Sex', widget=CheckboxFieldset(attrs={
+        'onchange': 'this.form.submit()'
```

Comment:
ahhhh, this is really nice - didn't realize the `attrs` would apply to all of them. kudos @bwhicks